js/repeater.js

```diff
@@ -116,7 +116,7 @@
 			self.resizeTimeout = setTimeout(function () {
 				self.resize();
 				self.$element.trigger('resized.fu.repeater');
-			}, 75);
+			}, 500);//any faster and you get weird catastrophic errors with the header of the list repeater
```

Comment:
So, this makes the UX inferior when resizing, because its slow and flickers when you resize.  I tried adding some code to `resize` to debounce calls to it, but it didn't fix the weird header resizing errors. So... I guess this has to be this way for now :(